CONTRIBUTING.md

```diff
@@ -1,270 +1,59 @@
-# Contributing to Hack Club
+# How To Contribute
 
-Contributions to Hack Club are very welcome, and strongly encouraged (even to
-this document)!
+This is a guide explaining the contribution workflow we use at Hack Club!
 
-## Doing it
+- [Find something you want to change](#find-something-to-change)
+- [If it is big, talk about it on Slack](#talk-about-it-on-slack)
+- [Create an issue if you want to proceed](#create-an-issue)
+- [Make a PR](#make-a-pr)
+- [Wait for review](#wait-for-review)
 
-We use a modified version of
-[GitHub Flow](https://guides.github.com/introduction/flow/) at Hack Club. The
-only difference is instead of deploying from a reviewed pull request, we merge
-first and deploy straight from master.
+## Find Something To Change
 
-Part of the GitHub Flow is submitting pull requests. See
-https://help.github.com/articles/using-pull-requests/ for a good overview of
-what pull requests are and how to use them. Internally, we use the _shared
-repository model_ for collaborative development, though we encourage external
-contributors to contribute through the _fork & pull model_.
+Usually, as a contributor you know what you want to work on. Perhaps something
```

Comment:
Also, there are 3 demographics that this is targeting:

- people that know what they want to do
- people that are looking for things to do who are normal contributors
- people that are looking for things to do who are beginner contributors

I would personally try to create that visual hierarchy that is discernible by just quickly glancing at it (i.e. using new lines)